lib/lifx/packets/stateInfo.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+var utils = require('../../lifx').utils;
+
+var Packet = {
+  size: 24
+};
+
+Packet.parseNanoseconds = function(buf) {
+  if (buf.length !== 8) {
+    throw new Error('Invalid length given for nanoseconds field');
+  }
+
+  var low = buf.readUInt32LE(0);
+  var high = buf.readUInt32LE(4);
+
+  return (high * 2**32 + low) / 1.0E9;
```

Comment:
The double `**` here is giving issues on the tests. Please also make this line a separate function to help understand what happens here.